include/mbedtls/aes.h

```diff
@@ -76,6 +76,9 @@ typedef struct mbedtls_aes_context {
     int MBEDTLS_PRIVATE(nr);                     /*!< The number of rounds. */
     size_t MBEDTLS_PRIVATE(rk_offset);           /*!< The offset in array elements to AES
                                                     round keys in the buffer. */
+#if defined(MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH) && !defined(MBEDTLS_PADLOCK_C)
+    uint32_t MBEDTLS_PRIVATE(buf)[44];
```

Comment:
A comment explaining why `buf` can be reduced to `44` elements might be nice. Something brief about AES-128 only having 10 round keys perhaps. Your call though.

Reply:
I have an extra commit to improve the definition of aes context but I got some test failure. So I planned to split that commit in a follow-up PR. Then I forgot to add some comments here. Will do it later.